gstreamer/Cargo.toml

```diff
@@ -37,6 +42,7 @@ embed-lgpl-docs = ["rustdoc-stripper"]
 purge-lgpl-docs = ["rustdoc-stripper"]
 dox = ["gstreamer-sys/dox", "glib/dox", "futures"]
 futures = ["futures-core"]
+serde-support = ["serde"]
```

Comment:
You don't need to add a feature but could directly use `serde` AFAIU

Reply:
That would be better indeed.